package.json

```diff
@@ -9,7 +9,8 @@
   },
   "scripts": {
     "build": "npm run nearley && tsc -b && npm run webpack && npx tailwindcss -i ./src/client/css/stylesheet.css -o ./public/css/stylesheet.css",
-    "lint": "npm run prettier-base --check && npm run eslint -- --max-warnings 0",
+    "lint": "npx prettier --check $(npm run --silent list-files); npx eslint --max-warnings 0 $(npm run --silent list-files)",
```

Comment:
lint was removed recently which broke the current master actions. I've added it back just for developers to use and updated the list-files paths